Extend/Model/Order.php

```diff
@@ -263,6 +263,7 @@ public static function getTranslatedStates()
             BackendService::TYPE_PROCESSING => Helper::translate('wd_order_status_purchased'),
             BackendService::TYPE_CANCELLED => Helper::translate('wd_order_status_cancelled'),
             BackendService::TYPE_REFUNDED => Helper::translate('wd_order_status_refunded'),
+            self::STATE_FAILED => Helper::translate('wd_order_status_failed'),
```

Comment:
Can u open an issue that the Order state is missing in the BackendService?

Reply:
Done 👍